tests/integration/test_database_client.py

```diff
@@ -67,3 +79,24 @@ def test_get_async_result_id(uuid, db_conn):
 def test_insert_files(files, db_conn):
     ids = client._insert_files(files)
     assert len(ids) == 2
+    assert isinstance(ids[0], int)
+    # tear down
+    for i in ids:
+        client._delete_file(i)
```

Comment:
Think about what happens if one of the asserts fails. Tear down should happen in every run.

Reply:
use try and finally to ensure teardown